cmd/composectl/cmd/check.go

```diff
@@ -266,7 +267,13 @@ func checkIfInstalled(ctx context.Context, appRefs []string, srcStorePath string
 				if s, err := reference.Parse(imageUri); err == nil {
 					taggedUri := s.Locator + ":" + (s.Digest().Encoded())[:7]
 					if !installedImages[taggedUri] {
-						missingImages = append(missingImages, imageUri)
+						// Check familiar name
+						if anyRef, err := docker.ParseAnyReference(imageUri); err == nil {
+							familiarRef := docker.FamiliarString(anyRef)
```

Comment:
Looks like this has been deprecated in favor or `reference.FamiliarString`

https://github.com/containerd/containerd/blob/v1.7.20/reference/docker/helpers_deprecated.go#L40

Reply:
It is not deprecated in the ctrd version we currently use https://github.com/containerd/containerd/blob/7c3aca7a610df76212171d200ca3811ff6096eb8/reference/docker/helpers.go#L43

Reply:
correct, but shouldn't we go ahead and the API that's not going away so we don't have to revisit this?

Reply:
Yes, provided that the API is available in the current version.
Is that API available in the currently used version of ctrd? I couldn't find it, so I just use the only API found in the given ctrd version.

Reply:
ah. n/m - just do this i guess